vnext/ReactUWP/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
+  <package id="Microsoft.UI.Xaml" version="2.2.190917002" targetFramework="native" />
```

Comment:
Seems like this is a nuget package that contains a framework package?

Reply:
yes. It's framework package

---
In reply to: [349762583](https://github.com/microsoft/react-native-windows/pull/3622#discussion_r349762583) [](ancestors = 349762583)